### PR TITLE
Update form labels and disclaimer

### PR DESCRIPTION
**Pull request by cehsu**
Resolves the following issues:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5660
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5662
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5663
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5667
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5668

# Disclaimer Update
**Burial**
-
screen shot 2017-11-03 at 11 02 03 am

**EDU (all edu forms use shared code for this)**
-
screen shot 2017-11-03 at 11 01 27 am

**HCA**
-
screen shot 2017-11-03 at 11 02 21 am

**Pension**
-
screen shot 2017-11-03 at 11 01 39 am

**Preneed**
-
screen shot 2017-11-03 at 11 33 25 am

screen shot 2017-11-03 at 11 33 46 am

# Labels Update
**Burials**
-
 “Start of service period” to “Service start date”
 “End of service period” to “Service end date”
 “Add Another” button (veteran name) to “Add Another Name”
-
localhost-3001-burials-and-memorials-application-530-military-history-service-periods 1

**EDU (all edu forms use shared code for this)**
-
 “Start of service period” to “Service start date”
 “End of service period” to “Service end date”
_
localhost-3001-education-apply-for-education-benefits-application-1990-military-history-service-periods 1

**HCA**
-
“Start of service period” to “Service start date”
 “Date of discharge” to “Service end date”
 “Character of discharge” to “Character of service”
_
localhost-3001-health-care-apply-application-military-service-service-information

 Capitalize Social Security in "Spouse’s social security number".
-
localhost-3001-health-care-apply-application-household-information-spouse-information

 "Add Another” button to “Add Another Insurance Policy”
-
localhost-3001-health-care-apply-application-military-service-service-information 1

 "Center/clinic” to “Center or clinic”
-
localhost-3001-health-care-apply-application-military-service-service-information 2

**Pension**
-
“Date entered active service” to “Service start date"
 “Date left active service” to “Service end date”
 “Add Another” button (military service) to “Add Another Service Period”
-

localhost-3001-pension-application-527ez-military-history-resume

 “Add Another” button (veteran name) to “Add Another Name”
-

localhost-3001-pension-application-527ez-military-history-resume 1

 “Add Another” button (dependent) to “Add Another Dependent”
-
localhost-3001-pension-application-527ez-household-dependents

### CODE REVIEW

File: src/js/common/schemaform/pages/applicantInformation.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default function applicantInformation(schema, options) {
     initialData: {},
     uiSchema: _.assign({
       'ui:order': fields,
-      'ui:description': <p>You aren’t required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>,
+      'ui:description': <p>You aren’t required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>,
```
Comment (jbalboni) [inline src/js/common/schemaform/pages/applicantInformation.jsx]: Should we use the ApplicantDescription component here?

File: src/js/pensions/helpers.jsx [hunk 2]
```diff
@@ -104,7 +104,7 @@ export const directDepositWarning = (
   </div>
 );
 
-export const applicantDescription = <p>You aren’t required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>;
+export const applicantDescription = <p>You aren’t required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>;
```
Comment (jbalboni) [inline src/js/pensions/helpers.jsx]: Same here, can we pull in the common component?

File: src/js/pre-need/config/form.js [hunk 3]
```diff
@@ -259,7 +262,7 @@ const formConfig = {
             applications: {
               items: {
                 'ui:title': claimantHeader,
-                'ui:description': <p>You aren't required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>,
+                'ui:description': <p>You aren't required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>,
```
Comment (jbalboni) [inline src/js/pre-need/config/form.js]: I think you can use the description component here, too?

**Review by jbalboni — APPROVED** [review]
Some minor comments, LGTM otherwise.

**Review by peggygannon — COMMENTED** [review]
@cehsu My only tweak is to remove the boldface from the word "all" in the disclaimer. 

Otherwise all other updates looks good!

File: src/js/common/schemaform/pages/applicantInformation.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/common/schemaform/pages/applicantInformation.jsx]: Thanks @jbalboni, excellent idea, change made.

File: src/js/pensions/helpers.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/pensions/helpers.jsx]: Thanks @jbalboni, excellent idea, change made.

File: src/js/pre-need/config/form.js (hunk 3 above)
Comment (cehsu) [reply src/js/pre-need/config/form.js]: Thanks @jbalboni, excellent idea, change made.

**Comment by cehsu** [comment]
Thanks @jbalboni @peggygannon, your recommendations have been integrated in the changes.

**Pull request merged**